app/client/cli/actor.go

```diff
@@ -86,18 +90,34 @@ A node can update relayChainIDs, serviceURI, and raise the stake amount with thi
 If the node is currently staked at X and you submit an update with new stake Y. Only Y-X will be subtracted from an account.
 
 If no changes are desired for the parameter, just enter the current param value just as before.`,
-		Args: cobra.ExactArgs(4), // REFACTOR(#150): <fromAddr> not being used at the moment. Update once a keybase is implemented.
+		Args: cobra.ExactArgs(4),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			// TODO(#150): update when we have keybase
-			pk, err := readEd25519PrivateKeyFromFile(privateKeyFilePath)
+			// Unpack CLI arguments
+			fromAddrHex := args[0]
+			fromAddr := crypto.AddressFromString(args[0])
+			amount := args[1]
+
+			// Open the keybase at the specified directory
```

Comment:
Not a debug anymore?

Reply:
@Olshansk no this now uses the prod keybase - however while we do the anonymous import the prod keybase at `$HOME/.pocket/keys` is being initialised with the 999 keys. Remove the import and it has no default keys and its using the normal prod ready DB

Reply:
💯